spec/factories/image_works.rb

```diff
@@ -22,6 +22,13 @@
       end
     end
 
+    factory :image_work_with_one_metadata_file do
+      after(:create) do |image_work, evaluator|
+
+        1.times { image_work.ordered_members << FactoryGirl.create(:external_metadata_file, user: evaluator.user) }
```

Comment:
@yulgit1 @drh-stanford This mirrors the approach undertaken for the specs implemented for the latest release of curation_concerns.  Firstly, I must clarify that, to the best of my understanding, the latest release of Hydra::PCDM ensures that *all* related resources managed using <pcdm:hasMember> (i. e. "members") are potentially ordered:

```
[...]
module Hydra::PCDM
  module PcdmBehavior
    extend ActiveSupport::Concern
    included do
      ordered_aggregation :members,
                          has_member_relation: Vocab::PCDMTerms.hasMember,
                          class_name: 'ActiveFedora::Base',
                          type_validator: type_validator,
                          through: :list_source
[...]
```
(https://github.com/projecthydra-labs/hydra-pcdm/blob/master/lib/hydra/pcdm/models/concerns/pcdm_behavior.rb#L5)

However, I believe that CurationConcerns prefers to test against this for cases where members are not explicitly ordered:
```
[...]
    factory :work_with_one_file do
      before(:create) do |work, evaluator|
        work.ordered_members << FactoryGirl.create(:file_set, user: evaluator.user, title: ['A Contained Generic File'], filename: 'filename.pdf')
      end
    end
[...]
```
(https://github.com/projecthydra-labs/curation_concerns/blob/master/spec/factories/generic_works.rb#L20)
....which is evaluated in the following specs:

```
[...]
  describe 'work associations' do
    let(:work) { create(:work_with_one_file) }
    subject { work.file_sets.first.reload }
    it 'belongs to works' do
      expect(subject.generic_works).to eq [work]
    end
  end
[...]
```
(https://github.com/projecthydra-labs/curation_concerns/blob/master/spec/models/file_set_spec.rb#L400)

```
[...]
  describe 'with a parent work' do
    let(:parent) { create(:work_with_one_file) }
    let(:parent_id) { parent.id }

    describe '#related_files' do
      let(:parent) { create(:work_with_files) }
      let(:sibling) { parent.file_sets.last }
      subject { parent.file_sets.first.reload }
      it 'returns related files, but not itself' do
        expect(subject.related_files).to eq([sibling])
        expect(sibling.reload.related_files).to eq([subject])
      end
    end
[...]
```
(https://github.com/projecthydra-labs/curation_concerns/blob/master/spec/models/file_set_spec.rb#L481)

From what I could determine, membership, rather than ordering, are scoped for these specs.

I have followed the CurationConcerns approach, based off of their factories.  If this appears to be less advantageous, I would certainly be open to the possibility of an alternative approach which preserves the scope of each test case.